src/library_int53.js

```diff
@@ -123,6 +123,19 @@ mergeInto(LibraryManager.library, {
     return (lo >>> 0) + (hi >>> 0) * 4294967296;
   },
 
+  // Given a signed i53 number, emit the low 32 bits. This is useful for
+  // calling into wasm when legalization is needed, and then instead of just
+  // doing f(num) you would do f(sendI53BitsLow(num), sendI53BitsHigh(num)).
+  $sendI53BitsLow: function(num) {
+    // TODO: assert on num being in the valid i53 range
+    return num | 0;
+  },
+  $sendI53BitsHigh: function(num) {
+    // TODO: assert on num being in the valid i53 range
+    num = Math.floor(num / 4294967296);
+    return num | 0;
+  },
```

Comment:
Why not make make a single I53 split function that returns the high/low pair?

Now that I think of that I remember there is an existing `splitI64` in parseTools.js.. maybe we don't want to keep both?

Reply:
yeah for now maybe just use `splitI64` and we can think of simplifying it later?

Reply:
I think that method is large and complex because (1) it handles non-i53 values and (2) it isn't clear on whether the input is signed or unsigned, and I think it tries to handle both. This PR's method assumes the input is a signed i53 which simplifies things a lot.

Reply:
For the purposes of the change in question isn't `splitI64` OK though?  If you want to add an alternative to `splitI64` maybe just add `splitI53` alongside it with a comment as to why its different?    Perhaps we don't need `splitI64` anymore at all?   I'd like to avoid duplicating its functionality though.. maybe this is good change to clean things up.

How about this:

1. Land the current wasmfs changes using the existing splitI64
2. Look into optimizing/removing/refactoring splitI64 as a followup

Reply:
Ok, if you prefer that I don't feel strongly. I changed it to that.

When we look at code size for WasmFS perhaps we can get back to this.